src/syscheckd/src/db/include/db.h

```diff
@@ -50,7 +51,8 @@ FIMDBErrorCode fim_db_init(int storage,
                            uint32_t min_sync_interval_time,
                            int value_limit,
                            bool sync_registry_enabled,
-                           int sync_thread_pool);
+                           int sync_thread_pool,
+                           int sync_queue_size);
 
```

Comment:
`sync_queue_size` is defined as an `int`, but stored as `long int` in the C structure. Use the same type in both places to avoid possible overflows. Change this to unsigned int